hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/client/HttpFSFileSystem.java

```diff
@@ -277,7 +283,7 @@ public enum Operation {
     GETSERVERDEFAULTS(HTTP_GET),
     CHECKACCESS(HTTP_GET), SETECPOLICY(HTTP_PUT), GETECPOLICY(HTTP_GET), UNSETECPOLICY(
         HTTP_POST), SATISFYSTORAGEPOLICY(HTTP_PUT), GETSNAPSHOTDIFFLISTING(HTTP_GET),
-    GETFILELINKSTATUS(HTTP_GET),
+    GETFILELINKSTATUS(HTTP_GET), GETSTATUS(HTTP_GET),
```

Comment:
```
GETFILELINKSTATUS(HTTP_GET), 
GETSTATUS(HTTP_GET)
```
?

Reply:
Thanks for your suggestion, I'll make the required code changes promptly